src/tpm/marshalling_tools.c

```diff
@@ -408,7 +596,15 @@ int create_ski_bytes(Ski input, uint8_t ** output, size_t * output_length)
                          &wk_pub_size,
                          wk_pub_offset,
                          &input.wk_priv,
-                         &wk_priv_data, &wk_priv_size, wk_priv_offset))
+                         &wk_priv_data,
+                         &wk_priv_size,
+                         wk_priv_offset,
+                         &input.policyBranch1,
+                         &p_branch_1_data,
+                         &p_branch_1_size,
+                         p_branch_1_offset,
+                         &input.policyBranch2,
+                         &p_branch_2_data, &p_branch_2_size, p_branch_2_offset))
```

Comment:
In the error handling code for this call, I think we need to add free() calls for `p_branch_1_data` and `p_branch2_data`.